library/spdm_crypt_lib/libspdm_crypt_aead.c

```diff
@@ -67,6 +115,10 @@ bool libspdm_aead_encryption(const spdm_version_number_t secured_message_version
     case SPDM_ALGORITHMS_AEAD_CIPHER_SUITE_AES_128_GCM:
     case SPDM_ALGORITHMS_AEAD_CIPHER_SUITE_AES_256_GCM:
 #if LIBSPDM_AEAD_GCM_SUPPORT
+        LIBSPDM_ASSERT((aead_cipher_suite != SPDM_ALGORITHMS_AEAD_CIPHER_SUITE_AES_128_GCM) ||
```

Comment:
What are these assertions for?

Reply:
The purpose of this check is to ensure if an algo is not supported in MACRO, the algo shall not be input from the caller.

The original version of assert is as follows:
```
#if !LIBSPDM_AEAD_AES_128_GCM_SUPPORT
    LIBSPDM_ASSERT(aead_cipher_suite != SPDM_ALGORITHMS_AEAD_CIPHER_SUITE_AES_128_GCM);
#endif
```

Reply:
Original version is good.

Reply:
Done. Change to original version.